docs/hacking/install.rst

```diff
@@ -122,23 +124,25 @@ system that Docker can be installed on:
 
 2. Download and run the
    `official NSQ image <https://hub.docker.com/r/nsqio/nsq/>`_,
-   the `official PostgreSQL image <https://hub.docker.com/_/postgres/>`_,
+   the `official PostgreSQL image <https://hub.docker.com/_/postgres/>`_, the
+   `official Redis image <https://hub.docker.com/_/redis/>`_,
    and our custom
    `Elasticsearch with ICU image <https://hub.docker.com/r/nickstenning/elasticsearch-icu/>`_:
 
    .. code-block:: bash
 
       docker run -d --name nsqd -p 4150:4150 -p 4151:4151 nsqio/nsq /nsqd
       docker run -d --name postgres -p 5432:5432 postgres
+      docker run -d --name redis -p 6379:6379 redis
```

Comment:
My feeling here is that soon enough the `alpine` tag will become the official `latest`. Let's wait for the upstream maintainers to make that call.